### PR TITLE
blockdev_inc_backup_merge_to_nonexist_bitmap:Merge bitmaps to a non-existed target

### PR DESCRIPTION
**Pull request by aliang123**
Merge bitmaps to a non-existed target

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:1984294

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_to_nonexist_bitmap.q35: PASS (42.82 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks

File: qemu/tests/blockdev_inc_backup_merge_to_nonexist_bitmap.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+def run(test, params, env):
+    """
+    Test for merging bitmaps to a non-exist bitmap target
+
+    test steps:
+        1. boot VM with a 2G data disk
+        2. format data disk and mount it, create a file
+        3. add two bitmaps to data disk
+        4. create a new file
+        5. merge two bitmaps merge to a non-exist bitmap:bitmap_tmp
```
Comment (zhencliu) [inline qemu/tests/blockdev_inc_backup_merge_to_nonexist_bitmap.py:53]: merge two bitmaps merge to a non-exist bitmap:bitmap_tmp -> merge two bitmaps to a non-exist bitmap:bitmap_tmp

**Comment by zhencliu** [comment]
Please update the test step, others look good to me

LGTM

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_to_nonexist_bitmap.q35: PASS (43.68 s)

File: qemu/tests/blockdev_inc_backup_merge_to_nonexist_bitmap.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_inc_backup_merge_to_nonexist_bitmap.py]: have updated, thanks zhenchao.

**Comment by ngu-niny** [comment]
1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_merge_to_nonexist_bitmap Finshed 2021-07-28 02:19:27 2021-07-28 02:21:45 PASS 138.543123

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**